project/src/components/app/app.tsx

```diff
@@ -1,5 +1,14 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import MainPage from '../../pages/mainPage/main';
+
+type Props = {
+  countOfAvailablePlaces: number;
```

Comment:
с точки зрения удобства чтения лучше называть перменные так:

```suggestion
  availablePlacesCount: number;
```